packages/terra-table/tests/jest/TableHeaderContent.test.jsx

```diff
@@ -19,9 +19,3 @@ it('should render table header content tag with descending sort indicator', () =
   const tableHeaderContent = shallow(tableHeaderContentTag);
   expect(tableHeaderContent).toMatchSnapshot();
 });
-
-it('should render table header content tag with maximum height fixed', () => {
```

Comment:
Why did we remove this test?

Reply:
Same as above

It was actually your recommendation. It is a stale test left by [this pull request removing max height tests](https://github.com/cerner/terra-core/pull/238)

Reply:
Opps forgot! 👍